hooks/useArtists.tsx

```diff
@@ -51,14 +52,18 @@ const useArtists = () => {
         ]
       : sortArtistsAlphabetically(artists);
 
+  const handleSetIsArtistBeingCreated = useCallback((value: boolean) => {
+    setIsArtistBeingCreated(value);
+  }, []);
```

Comment:
why do you need this `handle` function with `useCallback`?
- what happens if you directly export the `setIsArtistBeingCreated` function instead of wrapping it?